svthalia/concrexit
# Import events into newsletter

Closes #2162
Closes #3317



### Summary

- The board can now easily bulk-add events to the newsletter. This can probably use some nice defaults for the corresponding forms but I don't know what, @JeeVee11 probably does?
- The newsletter send page is now actually in the admin namespace
- A slight change in wording for the fine field in the admin
- Removed the `event` field for newsletter events, since this field was not used and kinda pointless (it was never used as far as I could see).

As you can see, apart from the headline feature there's a handful of refactors here as well.

I considered rewriting the newsletter app to use actual event references, but this is inflexible due to things like external events and would also require us to keep the current system alongside it to prevent breaking changes in our schema.

### How to test

1. Create a newsletter.
2. Click the nice pink `Import events` button.
3. Fill in the options.
4. Check that all the events you wanted appeared.
